### PR TITLE
Add 'autoresize' attribute.

### PR DESCRIPTION
**Pull request by LeoLei**
Working with ng-view and a parent container with percentage size, the chart can't resize properly.

Add a listener to window resize event and 'autoresize' attribute to resize the chart when the browser window size is changed.

### CODE REVIEW

File: angular-chart.js [hunk 1]
```diff
@@ -146,6 +147,11 @@
 
             chart = createChart(chartType, scope, elem);
           }
+
+          angular.element(window).on('resize', function () {
```
Comment (jtblin) [inline angular-chart.js]: It would be better to listen on the event only if the `autoresize` option has been set.

File: angular-chart.js [hunk 2]
```diff
@@ -146,6 +147,11 @@
...
+            if (isEmpty(scope.autoresize) || scope.autoresize == 'false') return;
```
Comment (jtblin) [inline angular-chart.js]: Prefer triple equality always `scope.autoresize === 'false'`.

**Comment by jtblin** [comment]
Thanks for the PR. chart.js should be responsive by default so it should be able to handle that case without specific code in this library, but if I understand correctly you're saying it happens because of using percentage in container dimensions? Would you be able to post a simple repro case (see jsbin template in readme) as if it's an issue in chart.js, I'd prefer submitting a PR to their repo to fix the root cause instead. I'd be fine with merging this PR in the meantime given the changes are small.

**Comment by LeoLei** [comment]
Open examples/autoresize.html in browser, then resize the window.The chart above with 'autoresize=true' will refresh automatically, the below one will still without any changing.

**Comment by jtblin** [comment]
The reason it didn't resize in the first place is because you set the `responsive` option to `false` in the first place. If you set the `responsive` option to `true` then it resizes properly without your change. Percentage size does not seem to be the problem either, you mentioned `ng-view`, could it be related to #29? I.e. the chart doesn't show up if it was created in a container with 0 height (e.g. hidden)?

**Comment by jtblin** [comment]
@LeoLei #29 is fixed now, could you check if this PR is still needed?

**Comment by LeoLei** [comment]
Sorry for the delay. I pulled the newest([bd70cdd](https://github.com/jtblin/angular-chart.js/commit/bd70cdd0784c98a781a0bb07d54521ddd4b040cb)) and set 'responsive' to true, it  does scale by width proportion when window size changes, but it's not what I want. Maybe it's just my special request, I hope that the chart can fill its parent container not just fit its width.

**Comment by jtblin** [comment]
@LeoLei ok makes sense about scaling when the container changes size but your `autoresize` PR only listens to the `window.resize` event so I don't think that would change anything (Chart.js already listens on this even and resizes the chart accordingly when `responsive` is `true`). I think what we need is to listen on the container size change but I don't think that's possible: https://developer.mozilla.org/en-US/docs/Web/Events/resize.

**Pull request closed without merging**